coralproject/talk
# Fixing mobile viewport width.

## What does this PR do?

Allows talk's sample articles to display appropriately in mobile.

## How do I test this PR?

1. Load talk in Chrome.
2. In the console, click "Toggle Device Toolbar"
![image](https://cloud.githubusercontent.com/assets/680687/22673132/21779588-ec8d-11e6-919e-b4bea5e8b9f7.png)
3. Fonts should be readable
